TheWarWithin/WarriorArms.lua

```diff
@@ -1517,8 +1516,20 @@ spec:RegisterAbilities( {
         debuff = "casting",
         readyTime = state.timeToInterrupt,
 
+        usable = function()
+            if not settings.spell_reflection_filter then return true end
+
+            local filters = class.reflectableFilters
+            local npcid = target.npcid
+            local t = debuff.casting
+
+            -- Only use on a reflectable spell targeted at the player.
+            return not not ( t.up and npcid and filters and filters[ npcid ] and filters[ npcid ][ t.v1 ] and UnitIsUnit( "player", t.caster .. "target" ) )
```

Comment:
`not not` ?

Reply:
That's a Lua-ism to ensure that the value is `true` or `false` and not `nil`. I read somewhere in the code that the `usable` function can return `nil` for a true result sometimes.

Reply:
State.lua, line 7261:
```
if usable == false then -- Have allowed nil return values for usable to be treated as usable before.
```